external/fv3fit/tests/reservoir/_regtest_outputs/test_model_adapter.test_adapter_predict.out

```diff
@@ -2,5 +2,5 @@
 Dimensions:  (x: 4, y: 4, z: 3)
 Dimensions without coordinates: x, y, z
 Data variables:
-    a        (x, y, z) float64 -1.967 0.5274 -1.969 -1.259 ... 5.788 4.426 4.439
-    b        (x, y, z) float64 -0.5784 -0.6441 -0.739 ... 4.685 4.086 2.695
+    a        (x, y, z) float64 1.486 -3.29 2.696 ... -0.3318 7.511 -0.06362
```

Comment:
These changed because the rng state was not pinned for generating the coefficient matrices and I switched to use a centralized helper function to grab the models during tests.